File: Modul3_C/PRAK302-2510817320010-PutriDeissytaHerliyana.c
#include <stdio.h>

int main() {
    int nilai;
    scanf("%d", &nilai);

    char huruf;

    if (nilai >= 80) {
        huruf = 'A';
    } else if (nilai >= 70 && nilai <= 79) {
        huruf = 'B';
    } else if (nilai >= 60 && nilai <= 69) {
        huruf = 'C';
    } else if (nilai >= 50 && nilai <= 59) {
        huruf = 'D';
    } else {
        huruf = 'E';
    }

    printf("%c\n", huruf);
    return 0;
}

File: Modul3_Pyton/PRAK302-2510817320010-PutriDeissytaHerliyana.py
nilai = int(input())

if nilai >= 80:
    huruf = "A"
elif 70 <= nilai <= 79:
    huruf = "B"
elif 60 <= nilai <= 69:
    huruf = "C"
elif 50 <= nilai <= 59:
    huruf = "D"
else:
    huruf = "E"

print(huruf)
